science/proxmark3-iceman/files/0001-Support-MacPorts.patch

```diff
@@ -0,0 +1,224 @@
+From 281f1efd7d9a3e0bd89ef3e0b2f004943817a5ab Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C4=B0lteri=C5=9F=20Ya=C4=9F=C4=B1ztegin=20Ero=C4=9Flu?=
+ <ilteris@asenkron.com.tr>
+Date: Wed, 7 Jul 2021 00:34:17 +0300
+Subject: [PATCH] Support MacPorts
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: İlteriş Yağıztegin Eroğlu <ilteris@asenkron.com.tr>
+---
+ Makefile.defs   |  5 +++-
+ client/Makefile | 80 +++++++++++++++++++++++++++++++++----------------
+ 2 files changed, 59 insertions(+), 26 deletions(-)
+
+diff --git a/Makefile.defs b/Makefile.defs
+index 40fb49b8b..5140ddfb3 100644
+--- a/Makefile.defs
++++ b/Makefile.defs
+@@ -59,7 +59,10 @@ else
+ endif
+ 
+ ifeq ($(USE_BREW),1)
+-  BREW_PREFIX = $(shell brew --prefix)
++  BREW_PREFIX = $(shell brew --prefix 2>/dev/null)
++  ifeq ($(strip $(BREW_PREFIX)),)
++    MACPORTS_PREFIX ?= /opt/local
++  endif
+ endif
```

Comment:
It looks like this block only takes effect if `USE_BREW` is equal to `1`. Is it? If so, seems like we should set it to `0` (e.g. in `build.args`). We don't ever want to run `brew` in something installed by MacPorts. Some user might have disregarded our instructions and might have installed both Homebrew and MacPorts. This is unsupported, but as much as possible, we want to not break if they do that. I guess there should be a separate `USE_MACPORTS` flag?